### DISCUSSION TITLE
Propose ideas !

### DISCUSSION THREAD

**Discussion by ledong1080** [zhblue/hustoj#802] [General]
Admin can add a function to limit the number of submissions for a problem  in contests.
Thanks.

**Comment by zhblue** [comment]
try add "noip" in the title of contest and set
```static  $OJ_OI_1_SOLUTION_ONLY=true; ```
the contest will use only the last submission of each problem, and will not tell the result until it's over.
the rule is used in Chinese OI contest used to be named as "NOIP".

**Reply by manni** [reply]
maybe he needs like 3 submissions...... @zhblue

**Comment by manni** [comment]
> 

This can only get solved by yourself, because this function is ***not*** required by many people now.

 Try to understand the code and modify them in ```/home/judge/src/web/submit.php``` you can add a mysql column to restrict the number of submissions in table ```problem.php```.
